louise.py

```diff
@@ -0,0 +1,24 @@
+balance = 2000
+min_wd = 50
+max_wd = 1000
+
+def ATM(withdrawal: int):
```

Comment:
Lovely function definition, but usually function names are in snake case or small letters